### PR TITLE
Fire sums for response time and session time metrics in addition to averages.

### PR DESCRIPTION
**Pull request by hodgestar**
Currently we only fire averages, but for many calculations (e.g. billing) it's useful to also have the total time, for which we need to fire sums.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: go/vumitools/middleware.py [hunk 1]
```diff
@@ -78,6 +78,13 @@ def is_optout_message(message):
         return message['helper_metadata'].get('optout', {}).get('optout')
 
 
+class TimeMetric(Metric):
+    """
+    A time-based metric that fires both sums and averages.
+    """
+    DEFAULT_AGGREGATORS = [AVG, SUM]
```
Comment (justinvdm) [inline go/vumitools/middleware.py:85]: I vaguely remember us only handling a single aggregator in some part of vumi-go, but probably not a part that affects this.

Comment (justinvdm) [inline go/vumitools/middleware.py:85]: :+1:

Comment (hodgestar) [inline go/vumitools/middleware.py:85]: Yep. That's in parts that go via conversation and account metrics.

**Pull request closed without merging**